package.json

```diff
@@ -61,14 +61,14 @@
 	},
 	"dependencies": {
 		"@types/node": "^13.13.15",
-		"@types/usb": "^1.5.1",
+		"@types/usb": "^1.5.2",
 		"usb": "^1.6.3"
 	},
 	"devDependencies": {
 		"@types/jest": "^23.3.14",
 		"coveralls": "^3.1.0",
 		"jest": "^26.4.2",
-		"speaker": "^0.5.2",
+		"speaker": "https://github.com/cjheath/node-speaker.git",
```

Comment:
could you explain a bit more what changes you need / made in your fork?

Reply:
c.f. https://github.com/TooTallNate/node-speaker/issues/142 - cloed but not fixed, I did this:
https://github.com/cjheath/node-speaker/commit/09377c3544742a32afaa4e8aeff8a6ada33f0205